phaseone/src/main/java/Utils.java

```diff
@@ -0,0 +1,10 @@
+public class Utils {
+
+    public Utils () {
```

Comment:
You have a Utils class... In Utils class there are going to be methods like openSession(), connectToDatabase(), forLoopTenTimes() etc.

please remove Utils() method